### ISSUE TITLE
Keep original maven repository as primary repository

### ISSUE THREAD

**Issue by stefanseifert** [Netcentric/maven-ext-repos-from-env#13]
thanks for this extension, it works very well! however, in my observation registering a private repository increased the duration of the "Build & Unit Testing" step in cloud manager by a magnitude (from <10 minutes to nearly 40 minutes).

the reason seems to be that now *everything* is downloaded from the private maven repository. and as the cloud manager does keep a local maven repository with release artifacts already download between execution steps (unlike e.g. travis where the cache can be used for this), it downloads the "whole internet" (tons of maven artifacts) on each run.

did you try to register the private repository as a secondary maven repository, so the primary source for downloading is still the adobe-configured repository, and the private repository is only used as fallback? is this possible with this extensions?

**Comment by kwin** [comment]
I am not sure how Maven calculates the order of repositories taking into account the different profile ids (like https://github.com/Netcentric/maven-ext-repos-from-env/blob/03782d5e94b0a0b09bcb550b44a89126078326fb/src/main/java/biz/netcentric/maven/extension/repofromenv/FromEnvReposConfigurationProcessor.java#L85). 
Probably we need to know the profile id in which the adobe repo is added (and also whether central is defined in a dedicated profile). I opened https://issues.apache.org/jira/browse/MNG-6994 to get some clarification here.
@ghenzler WDYT?

I also raised https://github.com/AdobeDocs/experience-manager-cloud-manager.en/issues/33 to get clarification how the Adobe mechanism of custom `settings.xml` does work.
